lib/grax/schema.ex

```diff
@@ -294,6 +298,22 @@ defmodule Grax.Schema do
 
   def schema?(_), do: false
 
+  @doc """
+  Returns all modules using `Grax.Schema`.
+  """
+  # ignore dialyzer assumes Grax.Schema.Registerable is always consolidated
+  @dialyzer {:nowarn_function, schemas: 0}
+  @spec schemas :: [module]
+  def schemas do
```

Comment:
Can we rename this to
```suggestion
  def known_schemas do
```

Reply:
sure, done